README.md

```diff
@@ -46,5 +46,205 @@ https://github.com/babel/babel/issues/2168
+| [`babel-traverse`](/packages/babel-traverse) | [![npm](https://img.shields.io/npm/v/babel-traverse.svg?maxAge=2592000)](https://www.npmjs.com/package/babel-traverse) | [![Dependency Status](https://david-dm.org/babel/babel.svg?path=packages/babel-traverse)](https://david-dm.org/babel/babel?path=packages/babel-traverse) | [![devDependency Status](https://david-dm.org/babel/babel/dev-status.svg?path=packages/babel-traverse)](https://david-dm.org/babel/babel?path=packages/babel-traverse#info=devDependencies) |
+| [`babel-generator`](/packages/babel-generator) | [![npm](https://img.shields.io/npm/v/babel-generator.svg?maxAge=2592000)](https://www.npmjs.com/package/babel-generator) | [![Dependency Status](https://david-dm.org/babel/babel.svg?path=packages/babel-generator)](https://david-dm.org/babel/babel?path=packages/babel-generator) | [![devDependency Status](https://david-dm.org/babel/babel/dev-status.svg?path=packages/babel-generator)](https://david-dm.org/babel/babel?path=packages/babel-generator#info=devDependencies) |
+
+`babel-core` is the Babel compiler itself; it exposes the `babel.transform` method, where `transformedCode = transform(src)`.
+
+The compiler can be broken down into 3 parts:
+- The parser: [`babylon`](https://github.com/babel/babylon) (moved to a seperate repo)
+- The transformer[s]: All the plugins/presets
+  - These all use [`babel-traverse`](/packages/babel-traverse) to traverse through the AST
+  - and [`babel-types`](/packages/babel-types) to validate, build, change AST nodes
+- The generator: [`babel-generator`](/packages/babel-generator)
+
+The flow goes like this:
+
+input string -> `babylon` parser -> `AST` -> transformer[s] -> `AST` -> `babel-generator` -> output string
+
+Check out the [`babel-handbook`](https://github.com/thejameskyle/babel-handbook/blob/master/translations/en/plugin-handbook.md#introduction) for more information on this.
+
+#### Other
+
+| Package | Version | Dependencies | DevDependencies |
+|--------|-------|------------|----------|
+| [`babel-cli`](/packages/babel-cli) | [![npm](https://img.shields.io/npm/v/babel-cli.svg?maxAge=2592000)](https://www.npmjs.com/package/babel-cli) | [![Dependency Status](https://david-dm.org/babel/babel.svg?path=packages/babel-cli)](https://david-dm.org/babel/babel?path=packages/babel-cli) | [![devDependency Status](https://david-dm.org/babel/babel/dev-status.svg?path=packages/babel-cli)](https://david-dm.org/babel/babel?path=packages/babel-cli#info=devDependencies) |
+| [`babel-types`](/packages/babel-types) | [![npm](https://img.shields.io/npm/v/babel-types.svg?maxAge=2592000)](https://www.npmjs.com/package/babel-types) | [![Dependency Status](https://david-dm.org/babel/babel.svg?path=packages/babel-types)](https://david-dm.org/babel/babel?path=packages/babel-types) | [![devDependency Status](https://david-dm.org/babel/babel/dev-status.svg?path=packages/babel-types)](https://david-dm.org/babel/babel?path=packages/babel-types#info=devDependencies) |
```

Comment:
You mention it in the core section above that's why I'm not sure if this might be also considered core?